### DISCUSSION TITLE
Welcome to Tridactyl discussions

### DISCUSSION THREAD

**Discussion by bovine3dom** [tridactyl/tridactyl#3076] [General]
## Welcome

If you have a question or an idea that you think doesn't quite make the bar for an issue, or could do with more discussion than is comfortable on Matrix, you can post it here. Or anything else remotely Tridactyl related. Oh, and feel free to help answer questions!

Feel free to introduce yourself in this thread, or not : )

You could mention your favourite and least favourite things about Tridactyl

**Comment by bovine3dom** [comment]
I'll start:

Hi, I'm Oliver / bovine3dom. I somewhat reluctantly started Tridactyl with my friend Colin a few years ago because Vimperator was dying and it looked like no one else was going to step in to replace it.

My favourite thing about Tridactyl is pressing `<C-6>` to switch between my most-recently used tabs, and my least-favourite thing is how if I type too quickly, text before the command line opens is lost.

**Reply by uhs-robert** [reply]
Hey, I'm Robert, and I just wanted to say thank you so much for making this 🙏🏻. I have been using Vim/NeoVim for maybe three years now, Vimium/Vimium C/Surfing Keys for the past two years, and recently developed HyprVim for Hyprland users like myself to have a Vim-like experience across the whole OS. So, I understand the itch to have a fully keyboard-driven workflow. But the web browsing experience has never felt good enough till I tried Tridactyl.

This add-on is the best web browsing Vim experience hands down, the only true comparison being Qutebrowser, which has its own unique set of issues. I love the `:command` mode implementation, it is just like Qutebrowser and allows for such flexibility and customizability. I have only been using Tridactyl for a week, but... my `tridactylrc` is _already_ over 300 lines and growing. The easter eggs like `credits` and `konamicode` are really nice touches that I love to stumble upon as well while I read the documentation.

Maybe the only thing I dislike is the absence of a whichkey system for chorded sequential commands like `;` to show the next sequential possibilities. If I have time, I'll take a look at the codebase and see if this is something that I can contribute to. Regardless, thanks again for making this. It really is quite exceptional!

**Reply by bovine3dom** [reply]
hi @uhs-robert , thanks for your kind words :) there's an ancient WIP for whichkey here https://github.com/tridactyl/tridactyl/pull/4355

it would be great if we could get it working and polished!

**Reply by uhs-robert** [reply]
I'll take a look! No promises, but hopefully we can get this implemented over time. 

Just had a baby a couple days ago so time is more limited than ever. But this is a feature I am motivated to make as I'd use it all the time (I love nested sequences of key binds).

**Reply by bovine3dom** [reply]
congratulations! obviously there's no rush :)
